backend/cmm_helpers.mli

```diff
@@ -1198,57 +1182,26 @@ val unboxed_int64_or_nativeint_array_set :
   expression
 
 (** {2 Getters and setters for unboxed int and float32 fields of mixed
-    blocks} *)
-
-(** The argument structure for getters is parallel to [get_field_computed]. *)
-
-val get_field_unboxed_int32 :
-  Asttypes.mutable_flag ->
-  block:expression ->
-  index:expression ->
-  Debuginfo.t ->
-  expression
-
-val get_field_unboxed_float32 :
-  Asttypes.mutable_flag ->
-  block:expression ->
-  index:expression ->
-  Debuginfo.t ->
-  expression
+    blocks} [immediate_or_pointer] is not needed as the layout is implied from the name,
```

Comment:
maybe add full break lines (including a white line so that ocamlformat doesn't break everything) so that the doc title (in brackets) is more distinguishable from the explanation text ?